server/service/scripts.go

```diff
@@ -13,9 +13,12 @@ import (
 
 	"github.com/docker/go-units"
 	"github.com/fleetdm/fleet/v4/pkg/scripts"
+	"github.com/fleetdm/fleet/v4/server/authz"
```

Comment:
All changes in this file are service methods moved from `ee` (premium) to the free tier.